README.md

```diff
@@ -115,38 +98,30 @@ export EXQLITE_SYSTEM_CFLAGS=-I/usr/local/include/sqlcipher
 export EXQLITE_SYSTEM_LDFLAGS=-L/usr/local/lib -lsqlcipher
 ```
 
-Once you have `exqlite` configured, you can use the `:key` option in the database config to enable encryption:
+Once you have `exqlite` build configured, you can use the `key` pragma to enable encryption:
 
 ```elixir
-config :exqlite, key: "super-secret'
+{:ok, conn} = Exqlite.open("sqlcipher.db")
+:ok = Exqlite.execute(conn, "pragma key='super-secret'")
```

Comment:
I've removed pragma helpers for a simpler API. Personally, I don't see much value in hiding the pragma statements from the user.

Reply:
My original intent behind the pragma helpers was to provide a more elixir / erlang interface to manipulate sqlite. Now with this setup, the caller will have to construct the strings to execute manually, which may or may not be a bad thing. I also wanted to be able to allow developers to configure exqlite via `config/*.exs` rather than having to make a factory module to produce and set configurations before utilizing the connection.

Now with removing the configurations, you are responsible for configuring your connection. Which is fine if `ecto_sqlite3` allows the pragmas to be set via configs.

```elixir
defmodule MyApp.ExqliteFactory do
  def build_conn do
    configs = Application.get_env(:my_app, Exqlite)

    {:ok, conn} = Exqlite.open(configs[:db])

    if configs[:key] do
      :ok = Exqlite.execute(conn, "pragma key='super-secret'")
    end

    # more connetion configuration setup here

    {:ok, conn}
  end
end
```

Reply:
With Ecto it would be something like

```config
config :my_app, MyApp.Repo,
  after_connect: fn conn ->
    :ok = Exqlite.execute(conn, "pragma key='super-secret'")
  end
```

Reply:
Makes sense to me. We'll need to document that as a solution in the README for those who want to alter the pragmas.